Cargo.toml

```diff
@@ -75,3 +75,7 @@ harness = false
 [[test]]
 name = "test-ptymaster-drop"
 path = "test/test_ptymaster_drop.rs"
+
+[[test]]
+name = "test-receive-timestampns"
```

Comment:
Why put it in its own test program, as opposed to the main "test" executable?  Only tests that require their own process should get their own test programs.  It's fine for the test to be a functional test; I don't think there's any reason to prefer a unit test instead.  But it shouldn't have its own test program.

Reply:
This is the CI result when the conditional compilation was turned off: https://cirrus-ci.com/build/5978723427024896

I think there are cases where the symbols are added, but it was not supported: https://cirrus-ci.com/task/5721153533640704?command=main#L510